dmutils/s3.py

```diff
@@ -59,6 +60,10 @@ def save(self, path, file_, acl='public-read', timestamp=None, download_filename
                 str(download_filename).encode("ascii", errors="ignore").decode(),
             )
 
+        content_type = self._get_mimetype(path)
+        if content_type is not None:
+            extra_kwargs['ContentType'] = content_type
+
```

Comment:
Hold on why would we want to upload documents with no mimetype?

Reply:
Why not? At this point they've passed fleep checks, but we're 'unable' to guess what the mimetype is

Reply:
We should be able to guess from the combination of the file extension and the zip header. Not having a mime type means when we serve it back to the user it probably won't prompt them to open it in a word processor.

Reply:
Also if anyone ever does add format validation to the bucket scanner it will need to know what to validate it _as_, having entries which are `None` will leave it puzzled.

Reply:
Is there any use/risk in having a human-recognisable value in there? `extra_kwargs['ContentType'] = content_type or 'undetectable'`

Reply:
i ... think i'd quite like it to continue throwing some sort of error if it can't figure out a mimetype for it, now Ben's fixed the problem thru the docker image. Might be better if the error were more obvious or happened earlier.

Reply:
@risicle Have removed this